build/publish/PublishNupkgToBlobFeed.targets

```diff
@@ -1,14 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <ItemGroup>
-    <NupkgsForPublishing Include="$(PackagesDirectory)/Microsoft.DotNet.Cli.Utils.*.nupkg" />
-    <NupkgsForPublishing Include="$(PackagesDirectory)/Microsoft.DotNet.MSBuildSdkResolver.*.nupkg" />
-  </ItemGroup>
 
   <Target Name="PublishNupkgToBlobFeed" Condition=" '$(OS)' == 'Windows_NT' And '$(Architecture)' == 'x64' " >
     <Error Condition="'$(BlobFeedCloudDropAccessToken)' == ''" Text="Missing property BlobFeedCloudDropAccessToken." />
     <Error Condition="'$(BlobFeedExpectedURL)' == ''" Text="Missing property BlobFeedExpectedURL." />
-    
+
+    <ItemGroup>
+      <NupkgsForPublishing Include="$(PackagesDirectory)/Microsoft.DotNet.Cli.Utils.*.nupkg" />
+      <NupkgsForPublishing Include="$(PackagesDirectory)/Microsoft.DotNet.MSBuildSdkResolver.*.nupkg">
+        <ManifestArtifactData>NonShipping=true</ManifestArtifactData>
+      </NupkgsForPublishing>
+    </ItemGroup>
```

Comment:
Why move this here? Can;t we just add the metadata to the line where this was before?

Reply:
It should have been here all along because the itemgroup only has meaning associated with this target. That being said, I do not have a strong opinion in this matter.

Reply:
I would strongly prefer that the diff does not have unrelated changes to the task at hand. Also, it is pretty common to set items that are static outside of a target. 

On that note, please squash once this gets down to one line.